trace_log/resources/application.properties

```diff
@@ -0,0 +1,54 @@
+#spring.main.sources=org.mycompany
```

Comment:
Please clean this file so that it only contains properties relevant to the trace_log